### PR TITLE
Add rsv-typer v0.1

### PR DESCRIPTION
**Pull request by DuyenLe2000**
Describe your pull request here
This PR adds rsv-typer to bioconda, a tool for genotyping RSV samples from nanopore reads
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by DuyenLe2000** [comment]
@BiocondaBot please fetch artifacts

**Comment by DuyenLe2000** [comment]
@BiocondaBot please add label

File: recipes/rsv-typer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{% set version = "0.1" %}
+
+package:
+  name: rsv-typer
+  version: {{ version }}
+
+source:
+  url: https://github.com/DiltheyLab/RSVTyper/archive/refs/heads/main.tar.gz
```
Comment (kbseah) [inline recipes/rsv-typer/meta.yaml:8]: The source URL should be stable, preferably pointing to a tagged version, see: https://bioconda.github.io/contributor/guidelines.html#stable-url

Could you please ask the repository owner to make a release on Github that can be pointed to here?

**Review by kbseah — CHANGES_REQUESTED** [review]
Thanks for your contribution! Please see my inline comment about the source URL

File: recipes/rsv-typer/meta.yaml (hunk 1 above)
Comment (DuyenLe2000) [reply recipes/rsv-typer/meta.yaml]: Hey, thank you so much for your review! I made a release and changed the URL

Comment (kbseah) [reply recipes/rsv-typer/meta.yaml]: one small thing, you can use the version variable `{{ version }}` in the source url too. these should also match the release tag in the file name (`0.1` vs `0.1.0`). have a look at other bioconda recipes to see the pattern. once that's fixed i think the recipe is good to go!

Comment (DuyenLe2000) [reply recipes/rsv-typer/meta.yaml]: Thank you! I changed those as well now

**Review by kbseah — APPROVED** [review]

**Pull request merged**